pkg/controller/account/ec2.go

```diff
@@ -64,7 +65,7 @@ func (r *ReconcileAccount) InitializeRegion(
 	reqLogger logr.Logger,
 	account *awsv1alpha1.Account,
 	region string,
-	ami string,
+	instanceInfo awsv1alpha1.InstanceInfo,
```

Comment:
I'm not sold that `InstanceInfo` is a good name for this.  At the beginning of my review it kind of made sense but when I'm looking at this now after being distracted it took a second look to realize what this was actually trying to pass in.

Maybe `InstanceParams`?  Idk if this sounds better though, because then I look at some of the other things passed in like the vCPUQuota and wonder why that wouldn't be passed in with the `InstanceParams`....

Maybe `AmiSpec` or something?

Reply:
+1